underfs/gcs/src/main/java/alluxio/underfs/gcs/GCSUnderFileSystem.java

```diff
@@ -64,7 +65,10 @@
   /** The name of the account owner. */
   private final String mAccountOwner;
 
-  /** The permission mode that the account owner has to the bucket. */
+  /**
```

Comment:
is this intended?

Reply:
Oh, changed back, thanks for finding it.